src/App.jsx

```diff
@@ -14,70 +15,54 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export function App() {
+  const [selectedGood, setValue] = useState('Jam');
+  const isSelected = goodName => goodName === selectedGood;
+  const selectGood = goodName => setValue(isSelected(goodName) ? '' : goodName);
+  const isEmptyState = () => selectedGood === '';
+  const heading = isEmptyState()
+    ? `No goods selected`
+    : `${selectedGood} is selected`;
+  const selectedGoodClassName = 'has-background-success-light';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {heading}
```

Comment:
here you can just check if selectedGood exists you show its name. In opposite way you show `No goods selected`. Just use ternary